### ISSUE TITLE
[a11y]: Missing tooltip in pagination select button

### ISSUE THREAD

**Issue by JKob207** [carbon-design-system/carbon#15563]
### Package

carbon-components-react

### Package version

10.58.12

### React version

v17.0.2

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checkes - Last Deployment

### Description

Problem: Missing tooltip in pagination select button
There should be a tooltip over select component in pagination or possibilty to send text for hovering over select dropdown.

### Reproduction/example

https://stackblitz.com/edit/github-zk1n5c?file=src%2FApp.jsx

### Steps to reproduce

Go to https://dataplatform.dev.cloud.ibm.com/governance/classifications
Go to classification details
Add 10 elements to any relationship
Hover over the page size dropdown

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tw15egan** [comment]
Hey there @JKob207 , can you please provide us a simplified reproduction on [StackBlitz](https://stackblitz.com/github/carbon-design-system/carbon/tree/main/examples/vite?file=README.md) so we can take a deeper look?

**Comment by JKob207** [comment]
> Hey there @JKob207 , can you please provide us a simplified reproduction on [StackBlitz](https://stackblitz.com/github/carbon-design-system/carbon/tree/main/examples/vite?file=README.md) so we can take a deeper look?

Hey, surely! There's a link to StackBlitz with reproduction and explaination of that problem: https://stackblitz.com/edit/github-zk1n5c?file=src%2FApp.jsx

**Comment by wkeese** [comment]
I don't see any reason to display a tooltip on that select, or selects in general.  I see the StackBlitz claims that:

> For accessbility reasons there should be a Tooltip as a wraper for this Select component to display a popup for an interactive element

But it doesn't explain why.   Can anyone site a rule that all selects (or all interactive elements) need a tooltip?

**Comment by tay1orjones** [comment]
@wkeese I agree - the computed accessible name for the select is `Page of 11 pages`, which makes sense. The visual layout inherently explains this for sighted users too - "{pageSelectValue} of 11 pages". A tooltip isn't necessary.
